src/terrain/terrain.cpp

```diff
@@ -608,13 +608,9 @@ Terrain::get_path_type(int xs, int ys, int zs, int xf, int yf, int zf) const {
     // so what is going on? Only god knows.
     // abs(_s - _f) returns zero or one depending on wether the final and
     // initial positions are the same. same as bool (_s != _f)
-    uint8_t x_diff = abs(xs - xf);
-    uint8_t y_diff = abs(ys - yf);
-    uint8_t z_diff = abs(zs - zf);
-
-    if (x_diff > 1 || y_diff > 1 || z_diff > 1){
-        LOG_ERROR(logging::terrain_logger, "diff is greater than one!!");
-    }
+    uint8_t x_diff = abs(xs - xf); // difference in x direction
+    uint8_t y_diff = abs(ys - yf); // difference in y direction
+    uint8_t z_diff = abs(zs - zf); // difference in z direction
```

Comment:
These comments are kinda useless

Reply:
my guy, my dude, you asked for comments. This functions has the most explicit comments of any in this repository, and you asked for more. Now you say there are too many.